src/studio/src/designer/backend/Controllers/TextController.cs

```diff
@@ -97,28 +98,30 @@ public IActionResult GetLanguages(string org, string app)
         [HttpPost]
         public IActionResult SaveResource([FromBody] dynamic jsonData, string id, string org, string app)
         {
-            id = id.Split('-')[0];
+            // TODO: Why is this method partially commented out?
+            // id = id.Split('-')[0];
             JObject json = jsonData;
 
-            JArray resources = json["resources"] as JArray;
-            string[] duplicateKeys = resources.GroupBy(obj => obj["id"]).Where(grp => grp.Count() > 1).Select(grp => grp.Key.ToString()).ToArray();
-            if (duplicateKeys.Length > 0)
-            {
-                return BadRequest($"Text keys must be unique. Please review keys: {string.Join(", ", duplicateKeys)}");
-            }
+            Console.WriteLine(org + app + id);
```

Comment:
@RubyJane88 I assume this should be reverted?-)

Reply:
This is something I added during the early POC for the frontend, just to get the basic functionality working, and it should be reverted 🙂

Reply:
done :)